taskcluster/scripts/write-dummy-secret.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
```

Comment:
This file is a copy of https://github.com/mozilla-mobile/fenix/blob/0d974fe2627c59d8f8b91a69acc42f51b8d14ead/taskcluster/scripts/write-dummy-secret.py with https://github.com/mozilla-mobile/fenix/pull/9206 applied.